### DISCUSSION TITLE
Use one Apislice but I need different baseQuery for Graphql and rest

### DISCUSSION THREAD

**Discussion by JoseAntpr** [reduxjs/redux-toolkit#2608] [Q&A] [answered]
Hi my question here if the recommendation is use only one API slice how I can configure the baseQuery to do calls for Rest and Graphql ??

**Comment by phryneas** [comment, marked as answer]
That seems you are interacting with two actually unrelated apis - in that case having two apis might actually be warranted - just be aware that mutating one of them will not invalidate the other.

**Reply by JoseAntpr** [reply]
Many thanks !! I think same as you but I needed to corroborate it
